src/pages/task/Metadata.tsx

```diff
@@ -125,8 +126,8 @@ export const Metadata: React.FC<Props> = ({ loading, task, error, taskId }) => {
         )}
         {/* Can only show the time running and eta if the task is running and 
       it has a baseTaskDuration to calculate the eta with */}
```

Comment:
Update comment.

Reply:
I deleted the comment instead because I think the code basically says the same thing